pkg/ingester/ingester_v2_test.go

```diff
@@ -1815,6 +1815,48 @@ func TestIngester_flushing(t *testing.T) {
 				m.AssertNumberOfCalls(t, "Sync", 1)
 			},
 		},
+
+		"noBlockSpanningOverADayOnFlush": {
```

Comment:
> noBlockSpanningOverADayOnFlush

What does "noBlock" mean here? Also could you write it in plain english, plese?

Reply:
I will update it :P